internal/proto/networkfence.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.30.0
-// 	protoc        v3.19.6
```

Comment:
You are using an old version to re-generate these files, please use protoc v3.19.6